### PR TITLE
Converting toolbar to TS

### PR DESCRIPTION
**Pull request by brumik**

### CODE REVIEW

**Comment by h-kataria** [comment]
@brumik ran into error with `npm start`

```
ERROR in /Users/hkataria/dev/tower-analytics-frontend/src/Components/Toolbar/Toolbar.tsx
./src/Components/Toolbar/Toolbar.tsx 21:9-21
[tsl] ERROR in /Users/hkataria/dev/tower-analytics-frontend/src/Components/Toolbar/Toolbar.tsx(21,10)
      TS2305: Module '"../../Utilities/qs"' has no exported member 'handleSearch'.
ts-loader-default_e3b0c44298fc1c14
 @ ./src/Components/Toolbar/index.tsx 1:0-32 2:15-22
 @ ./src/Containers/JobExplorer/JobExplorer.js 21:0-58 72:179-196
 @ ./src/Routes.js 23:11-57
 @ ./src/App.js 3:0-34 38:44-50
 @ ./src/AppEntry.js 5:0-24 14:38-41
 @ container entry ./RootApp[0]

webpack 5.38.1 compiled with 1 error in 31136 ms
ℹ ｢wdm｣: Failed to compile.

```

**Comment by knightsean00** [comment]
> @brumik ran into error with `npm start`
> 
> ```
> ERROR in /Users/hkataria/dev/tower-analytics-frontend/src/Components/Toolbar/Toolbar.tsx
> ./src/Components/Toolbar/Toolbar.tsx 21:9-21
> [tsl] ERROR in /Users/hkataria/dev/tower-analytics-frontend/src/Components/Toolbar/Toolbar.tsx(21,10)
>       TS2305: Module '"../../Utilities/qs"' has no exported member 'handleSearch'.
> ts-loader-default_e3b0c44298fc1c14
>  @ ./src/Components/Toolbar/index.tsx 1:0-32 2:15-22
>  @ ./src/Containers/JobExplorer/JobExplorer.js 21:0-58 72:179-196
>  @ ./src/Routes.js 23:11-57
>  @ ./src/App.js 3:0-34 38:44-50
>  @ ./src/AppEntry.js 5:0-24 14:38-41
>  @ container entry ./RootApp[0]
> 
> webpack 5.38.1 compiled with 1 error in 31136 ms
> ℹ ｢wdm｣: Failed to compile.
> ```

Mine seems to work just fine

**Comment by brumik** [comment]
@h-kataria why would you use `npm start`? Aren't you running the project with `npm run start:beta`?

**Comment by h-kataria** [comment]
> @h-kataria why would you use `npm start`? Aren't you running the project with `npm run start:beta`?

sorry i meant `make start`

**Comment by h-kataria** [comment]
Screen Shot 2021-08-10 at 9 50 03 AM
@brumik ^

File: src/Components/Toolbar/constants.ts [hunk 1]
```diff
@@ -3,22 +3,19 @@ export interface OptionsForCategories {
...
     hasChips: boolean;
-    isSingle?: boolean;
     placeholder?: string;
   };
 }
 
 export const optionsForCategories: OptionsForCategories = {
   status: {
     type: 'select',
-    isSingle: false,
```
Comment (h-kataria) [inline src/Components/Toolbar/constants.ts:14]: @brumik where is this property being set for all these objects with ts conversion, couldn't find that being set

**Review by h-kataria — APPROVED** [review]
looks good and seems to be working well in UI

File: src/Components/Toolbar/constants.ts (hunk 1 above)
Comment (brumik) [reply src/Components/Toolbar/constants.ts:14]: I realized we don't need it from the types. We have an array of values if it is false and we have a non array value if it is single. So telling explicitly is redundant.

**Pull request merged**